.github/workflows/backwards_compatibility_tests_workflow.yml

```diff
@@ -0,0 +1,49 @@
+name: Backwards Compatibility Tests SearchRelevance
```

Comment:
awesome, now we do have BWC for search-relevance. Thank you for adding this extension, I hope with this we can catch incompatible changes to index schema